NBug/NBug.csproj

```diff
@@ -10,9 +10,9 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>NBug</RootNamespace>
     <AssemblyName>NBug</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>

```

Comment:
4.6.1 in GE

Reply:
I think it is safe to move it to the same version as the rest of the app.

Reply:
It should of course be possible, but nothing in this change required upgrading past 4.5. I would recommend implementing the unification as a separate change if it's desired.